### PR TITLE
fix: fix previous structure

### PR DESCRIPTION
**Pull request by elenacliu**
I got `pretrain` and `finetune` process mixed up, and previously I pretrained the models using finetune model structure, which might lead to worse results. The fix code is in process and may be finished in a few days.

### CODE REVIEW

**Comment by elenacliu** [comment]
@jemmyshin can you review this code for me?

**Comment by bwanglzu** [comment]
this is great! we would like to bring this into Finetuner! in case @elenacliu do not have time, we could take over ! great work @elenacliu ! @jemmyshin please proceed

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 -f https://download.pytorch.org/whl/torch_stable.html
+finetuner==0.4.1
```
Comment (jemmyshin) [inline requirements.txt:2]: we can loose the restriction on `finetuner` version

**Review by jemmyshin — CHANGES_REQUESTED** [review]
Also please make sure that `finetuner.py` is runnable on the latest verison of `finetuner`

**Pull request merged**